api4/file.go

```diff
@@ -120,7 +123,14 @@ func uploadFile(c *Context, w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		resStruct, appErr = c.App.UploadMultipartFiles(FILE_TEAM_ID, channelId, c.Session.UserId, m.File["files"], m.Value["client_ids"])
+		resStruct, appErr = c.App.UploadMultipartFiles(
```

Comment:
If multiple files are uploaded as part of the same request, am I correct in understanding that they will all have the same `create_at` timestamp? Looking at https://github.com/mattermost/mattermost-server/pull/7022, it seems like the resulting order will be non-deterministic.

Should we consider having the client manage a counter to make this ordering explicit in all cases?

Reply:
That's right.  I have submitted change to mattermost-redux (https://github.com/mattermost/mattermost-redux/pull/582) to cover both RN and webapp sorting of files, and also another to webapp (https://github.com/mattermost/mattermost-webapp/pull/1483) primarily on file previews and minor clean up.  No similar change to RN since we're selecting file one at a time only.